package.json

```diff
@@ -146,7 +152,7 @@
 		"progress": "2.0.3",
 		"react": "16.9.0",
 		"react-dom": "16.9.0",
-		"react-native": "0.60.0",
+		"react-native": "jtreanor/react-native#v0.60.0-patched",
```

Comment:
Just noting that in `master`, we use RN 0.61.x. I'm not sure if it changes anything.

Reply:
Yes, it's quite a recent upgrade, we'll need to upgrade for this feature branch as well. Given the [amount of changes](https://github.com/wordpress-mobile/gutenberg-mobile/pull/1450) related to this in gutenberg-mobile it deserves its own PR imo.